src/module-3/online-store/index.html

```diff
@@ -1,16 +1,524 @@
 <!DOCTYPE html>
-<html lang="en">
-
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Online Store</title>
-  <link rel="stylesheet" href="style.css">
-</head>
-
-<body>
-
-</body>
-
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300;400;600&display=swap" rel="stylesheet">
+    <link rel="stylesheet" type="text/css" href="css/styles.css" />
+    <link rel="shortcut icon" type="image/png" href="img/favicon.ico">
+    <title>Online Store</title>
+  </head>
+  <body>
+  <main class="overlap-main">
+    <header class="header">
+      <img class="header__logo" src="img/logo.png" />
+      <h1 class="header__title">Online Store</h1>
+    </header>
+    <nav class="nav">
+      <a href="#"><img class="nav__ico--home" src="img/home.svg" /></a>
+      <img class="nav__ico--right" src="img/chevrons-right-1.svg" />
+      <a href="#" class="nav__link-1">eCommerce</a>
+      <img class="nav__ico--right" src="img/chevrons-right-2.svg" />
+      <div class="nav__link-2">Electronics</div>
+    </nav>
+    <div class="flex-row-main">
+      <aside class="flex-col-right">
+        <div class="category">
+          <div class="filters montserrat-normal-heavy-metal-18px">Filters</div>
```

Comment:
you may use a mixin instead of such a large class `montserrat-normal-heavy-metal-18px`